app/helpers/searches/paths_helper.rb

```diff
@@ -22,11 +22,11 @@ def path_for_filterable_search(search, search_params, extra_params = {})
   end
 
   def path_for_i14y_search(search, search_params, extra_params = {})
-    i14y_params = search_params.slice(:affiliate, :m)
+    i14y_params = search_params.slice(:affiliate, :m, :dc)
     i14y_params[:query] = search.query
     i14y_params.merge! extract_current_search_filter_params(search)
     i14y_params.merge! extra_params
-    search_path i14y_params
+    url_for i14y_params
```

Comment:
[TIL](https://apidock.com/rails/ActionView/Helpers/UrlHelper/url_for#473-Use-the-current-URL-with-changes), wow.

Reply:
Riiiiight? You and me both.

Reply:
@viclim ,thanks for the work you did on this story. I didn't actually mean to steal it, but in the process of poking the code to determine how it *should* work, I figured out how it *could* work.

Reply:
@nickmarden , the return value is just the path, i.e.:
`/search/docs?affiliate=764544198&query=test+query&dc=1&sort_by=date&tbs=m`